data/json/recipes/weapon/ranged.json

```diff
@@ -260,13 +260,13 @@
   },
   {
     "type": "recipe",
-    "result": "slam_shotgun",
+    "result": "four_winds_shotgun",
```

Comment:
You'd need to add a recipe obsoletion to json/recipes/recipe_obsolete.json to change the ID:
```
{
    "type": "recipe",
    "result": "slam_shotgun",
    "obsolete": true
  },
```

Reply:
Where do I put these things?